modules/oe_content_tender/src/TenderNodeWrapperInterface.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_tender;
+
+use Drupal\Component\Render\MarkupInterface;
+use Drupal\Core\Datetime\DrupalDateTime;
+
+/**
+ * Interface for the tender entity wrapper.
+ */
+interface TenderNodeWrapperInterface {
+
+  /**
+   * Tender isn't available. Is used when opening date isn't defined.
+   */
+  const TENDER_STATUS_NOT_AVAILABLE = 'n/a';
+
+  /**
+   * Tender is upcoming.
+   */
+  const TENDER_STATUS_UPCOMING = 'upcoming';
+
+  /**
+   * Tender is open.
+   */
+  const TENDER_STATUS_OPEN = 'open';
+
+  /**
+   * Tenser is closed.
+   */
+  const TENDER_STATUS_CLOSED = 'closed';
+
+  /**
+   * Check whether the tender status is 'open'.
+   *
+   * @return bool
+   *   Whereas the tender status is 'open'.
+   */
+  public function isOpen(): bool;
+
+  /**
+   * Check whether the tender status is 'upcoming'.
+   *
+   * @return bool
+   *   Whereas the tender status is 'upcoming'.
+   */
+  public function isUpcoming(): bool;
+
+  /**
+   * Check whether the tender status is 'closed'.
+   *
+   * @return bool
+   *   Whereas the tender status is 'closed'.
+   */
+  public function isClosed(): bool;
+
+  /**
+   * Check whether the tender status is not available.
+   *
+   * @return bool
+   *   Whereas the tender status is not available.
+   */
+  public function isNotAvailable(): bool;
```

Comment:
Given that the status field is optional (can't recall right now) let's change this into:
```suggestion
  public function hasStatus(): bool;
```